### PR TITLE
Add depth control preset option

### PR DESCRIPTION
**Pull request by mdhorn**
Changes proposed in this pull request:
- add depth control preset to dynamic reconfigure options for r200
- add individual depth control options to dynamic reconfigure options for r200
- glue code to update individual options when preset select; also invalidate preset when individual option is changed

### CODE REVIEW

**Comment by anelliot** [comment]
Looks good to me. √

File: realsense_camera/src/r200_nodelet.cpp [hunk 1]
```diff
@@ -114,11 +116,150 @@ namespace realsense_camera
   }
 
   /*
+   * Change the Depth Control Preset
+   */
+  void R200Nodelet::setDynamicReconfigDepthControlPreset(int preset)
+  {
+    // There is not C++ API for dynamic reconfig so we need to use a system call
```
Comment (reaganlo) [inline realsense_camera/src/r200_nodelet.cpp]: Typo - "no" instead of "not" :)

File: realsense_camera/src/r200_nodelet.cpp [hunk 2]
```diff
@@ -114,11 +116,150 @@ namespace realsense_camera
...
+   * Change the Depth Control Individual values
+   * GET ALL of the DC options from librealsense
+   * Call dynamic reconfig and set all 10 values as a set
+   */
+  std::string R200Nodelet::setDynamicReconfigDepthControlIndividuals()
+  {
+    std::string current_dc;
+    std::string option_value;
+
+    // There is not C++ API for dynamic reconfig so we need to use a system call
```
Comment (reaganlo) [inline realsense_camera/src/r200_nodelet.cpp]: Typo - "no" instead of "not" :)

**Review by reaganlo — APPROVED** [review]
Phew.. Thats some code that you have written.. :)

**Pull request merged**